src/CheckBoxFilter.webmodeler.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, createElement } from "react";
+import * as classNames from "classnames";
+
+import { Alert } from "./components/Alert";
+import { CheckboxFilter } from "./components/CheckBoxFilter";
+import { ContainerProps } from "./components/CheckBoxFilterContainer";
+import { Utils, parseStyle } from "./utils/ContainerUtils";
+
+// tslint:disable-next-line class-name
+export class preview extends Component<ContainerProps, {}> {
+    constructor(props: ContainerProps) {
```

Comment:
Add space before constructor

Reply:
Not necessary since we've not private member variables.